From a924118799c664f9ca93a6783a699cd982b52764 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Mon, 4 Apr 2016 14:27:17 -0700
Subject: [PATCH 080/101] ARM: bcm2835: Give HDMI a HPD GPIO on the pi3
Git-commit: 015836a799c518af6a04d7b5b8b01b137789d09a
Patch-mainline: Queued in subsystem maintainer repository
Git-repo: https://github.com/anholt/linux.git
References: FATE#319481

This is the wrong GPIO, but it means vc4 probes and turns on hdmi.

Signed-off-by: Alexander Graf <agraf@suse.de>

---
 arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts b/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts
index 223793d..d549791 100644
--- a/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts
+++ b/arch/arm64/boot/dts/broadcom/bcm2837-rpi-3-b.dts
@@ -27,3 +27,7 @@
 &uart1 {
 	status = "okay";
 };
+
+&hdmi {
+	hpd-gpios = <&gpio 46 GPIO_ACTIVE_LOW>;
+};
-- 
1.8.5.6

